File: RB101/study_guide/live_assesment/small_problems/easy_1/banner.rb
 def print_in_box(string)
  banner_line = ''
  empty_line = ''
  string.size.times { empty_line << ' '}
  string.size.times { banner_line << '-'}
  string.prepend('| ')
  string << ' |'

  puts top_line = '+-' + banner_line + '-+'
  puts '| ' + empty_line + ' |'
  puts string
  puts '| ' + empty_line + ' |'

  puts top_line
 end
 
 
 
 
 puts print_in_box('To boldly go where no one has gone before.')

File: JS210/exersies/easy2/banner.js
function logInBox(string, length) {

  if (length !== undefined) {
    string = string.slice(0, length);
  }

  let topRow = '+-' + '-'.repeat(string.length) + '-+';
  let rowTwo = '| ' + ' '.repeat(string.length) + ' |';

  console.log(topRow);
  console.log(rowTwo);
  console.log('| ' + string + ' |');
  console.log(rowTwo);
  console.log(topRow);
}




logInBox('To boldly go where no one has gone before.');
logInBox('');